sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/__init__.py

```diff
@@ -7,7 +7,7 @@
     CertificatePolicyAction,
     KeyCurveName,
```

Comment:
These should be `CertificateKeyCurveName`, `CertificateKeyType`, and `CertificateKeyUsage`.

Reply:
We talked about this in the meeting yesterday, since Python doesn't have a problem sharing enum names with keys we're keeping these without the Certificate prefix